.travis.yml

```diff
@@ -2,32 +2,32 @@
 sudo: false
 language: python
 env:
-  - TOXENV=py27-django18-sqlite
-  - TOXENV=py27-django18-mysql
-  - TOXENV=py27-django18-postgres
+  - TOXENV=py27-django18-sqlite PYTHONPATH=$HOME/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
```

Comment:
hi @julen what this PR does is create a python env outside of tox - ie in the default travis env - and then make this available to the tox venvs.

there are some implications to the changes that we are making to speed up test runs though (although not entirely from this PR)

I will write up something more substantial, but worth noting:

- python libs are only installed when you run uncached - so if you want to update upstream you need to clear cache
- static resources are only cooked into cache - so again you need to clear cache to get changes - this may error anyway without clearing cache - not sure
- when pushing to master you should clear and test PR uncached to ensure it works with latest upstream etc